.github/workflows/scripts/detect_ts_ignore.py

```
@@ -27,29 +67,29 @@ def check_ts_ignore(files: list[str]) -> int:
+                            TS_IGNORE_PATTERN,
                             line.strip(),
-                        )
-                        ts_ignore_found = True
-        except FileNotFoundError:
-            logging.warning("File not found: %s", file)
-        except OSError:
-            logging.exception("Could not read %s", file)
+                        ):
+                            print(
+                                f"❌ Error: '@ts-ignore' found in {file} "
+                                f"at line {line_num}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)